Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -383,6 +404,35 @@ public override void Render(IRenderContext rc, PlotModel model)
                         rc.DrawEllipse(ellipseRect, fillColor, OxyColors.Undefined, 0);
                     }
                 }
+
+                if (!this.ShowMeanAsDot && !double.IsNaN(item.Mean))
+                {
+                    // Draw the median line
+                    var meanLeft = this.Transform(item.X - halfBoxWidth, item.Mean);
+                    var meanRight = this.Transform(item.X + halfBoxWidth, item.Mean);
+                    rc.DrawClippedLine(
+                        clippingRect,
+                        new[] { meanLeft, meanRight },
+                        0,
+                        strokeColor,
+                        this.StrokeThickness * this.MeanThickness,
+                        LineStyle.Dash.GetDashArray(),
+                        LineJoin.Miter,
+                        true);
+                }
+                else if (!double.IsNaN(item.Mean))
```

Comment:
I'm not a big fan of "over"-nesting everything, but it's your project so whatever codestyle you like better is fine to me. so should I change this?